src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/components/reference/inputs/EceWorkExperienceReferenceAssessment.vue

```diff
@@ -0,0 +1,302 @@
+<template>
+  <v-row>
+    <v-col cols="12" md="12" lg="12" xl="12">
+      <h3>Competencies Assessment</h3>
+      <div role="doc-subtitle">
+        Tell us your opinion on the competency of the applicant in the following areas. This should be based on your personal observations.
+      </div>
+
+      <v-row class="mt-5">
+        <v-col cols="12" md="8" lg="6" xl="4">
+          <RadioWithAdditionalOption
+            :items="likertScaleRadio"
+            :trigger-values="likertScaleTrigger.map((item) => item.value)"
```

Comment:
I was thinking we could simplify this piece. 

We we just need an array of strings ['word1','word2']

so we could just pass this in instead of mapping it to a new array.

Reply:
Agreed. Done